src/main/java/bio/terra/service/job/StairwayLoggingHooks.java

```diff
@@ -49,6 +52,7 @@ public HookAction startFlight(FlightContext context) {
     MDC.put(FLIGHT_ID_KEY, context.getFlightId());
     MDC.put(FLIGHT_CLASS_KEY, context.getFlightClassName());
     MDC.put(FLIGHT_OPERATION_KEY, FLIGHT_OPERATION_START);
+    MDC.put(FLIGHT_PARENT_ID_KEY, getParentFlightId(context));
```

Comment:
I'm wondering if we should not log this at all if there is no parent (might make sense to keep the logs a little smaller?)

Reply:
Good idea!  That thought crossed my mind, I will add it conditionally.